src/components/Modals/CourseMenu.vue

```diff
@@ -144,10 +184,14 @@ export default {
       height: 16px;
     }
   }
-
   &-colors {
     position: absolute;
     right: -9rem;
   }
+  &-editCredits {
+    position: absolute;
+    width: 2.7rem;
```

Comment:
Good practice to use more precise rem values - so something like 2.5 rem or 2.75 rem that match px values (since 16 px = 1 rem)